src/components/common/modal/ModalPaymentStatus.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { View } from 'react-native'
+import Text from '../view/Text'
+
+const statusLabel = {
+  sendpending: 'Payment Pending...',
+  sendcancelled: 'Payment Cancelled',
+  sendcompleted: 'Payment Completed!',
+}
+
+const PaymentStatus = ({ item }) =>
+  item.displayType in statusLabel && (
+    <View style={{ marginVertical: '10vh' }}>
```

Comment:
why not just use full height (as the same height that uses when there is an image) and center the text?

Reply:
Updated